app/controllers/posts_controller.rb

```diff
@@ -1,12 +1,12 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = User.includes(:posts).find(params[:user_id])
```

Comment:
[REQUIRED] As instructed, kindly eager load your comments. In fact, the comments have more risk of falling into the n + 1 problem. Because for each post you load from users you will need to load an unknown amount of comments. Kindly adjust your code like this: `@posts = @user.posts.includes(:comments)`

Reply:
Thanks!! I didn't really know how to implement eager load on this code.